tensorboard/backend/event_processing/event_accumulator_test.py

```diff
@@ -725,13 +725,11 @@ def _writeMetadata(self, logdir, summary_metadata, nonce=''):
     writer.add_summary(summary.SerializeToString())
     writer.close()
 
-
   def testSummaryMetadata(self):
     logdir = self.get_temp_dir()
     summary_metadata = tf.SummaryMetadata(
-        display_name='current tagee',
-        summary_description='no',
-        plugin_data=[tf.SummaryMetadata.PluginData(plugin_name='outlet')])
+        display_name='current tagee', summary_description='no')
+    summary_metadata.plugin_data.plugin_name = 'outlet'
```

Comment:
As in the internal change, there's no need to introduce a statement context here:
```py
summary_metadata = tf.SummaryMetadata(
    display_name='…',
    summary_description='…',
    plugin_data=tf.SummaryMetadata.PluginData(plugin_name='…'))
```
Likewise throughout this PR.